test/easyconfigs/easyconfigs.py

```diff
@@ -820,6 +821,20 @@ def template_easyconfig_test(self, spec):
     error_msg = "Found use of '$root', not compatible with modules in Lua syntax, use '%%(installdir)s' instead: %s"
     self.assertFalse(res, error_msg % res)
 
+    # check for redefined easyconfig parameters, there should be none...
+    param_def_regex = re.compile('^(?P<key>[a-z_]+)\s*=', re.M)
```

Comment:
@boegel some (local_*) parameters also contain digits and capital letters, so shouldn't this be:
```
param_def_regex = re.compile(r'^(?P<key>\w+)\s*=', re.M)
```

Reply:
Good point, fixed in 6daa654